src/DoubleLinkedList.sol

```diff
@@ -79,12 +77,11 @@ library DoubleLinkedList {
     /// @param id The address of the account.
     function remove(List storage list, address id) internal {
         Account memory account = list.accounts[id];
+        if (id == address(0)) revert AddressIsZero();
```

Comment:
useless as the value of account zero cannot be something else than zero

Reply:
I don't think it is useless as if you remove the account of address(0). You open the loop and you cannot find tail and head easily.

Reply:
I will do the changes as expected for the insertSorted function.

Reply:
yes you are right.

Reply:
I think that it was an error